.circleci/run_docker_build.sh

```diff
@@ -54,15 +55,15 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artifacts.
 conda clean --lock
 
-conda update conda conda-build
-conda install conda-build-all
-conda install conda-forge-build-setup
+source activate root
```

Comment:
root is superfluous here.  ``source activate`` with no argument will activate root.  I prefer it also because conda 4.4 calls the root env "base," although referring to it as "root" still works, too.